diff --git a/page_title.tokens.inc b/page_title.tokens.inc
index 51970bc..5a3fc86 100644
--- a/page_title.tokens.inc
+++ b/page_title.tokens.inc
@@ -13,6 +13,10 @@ function page_title_token_info() {
     'name' => t('Page Title'),
     'description' => t('The title of the current page, as processed by the Page Title module.'),
   );
+  $info['tokens']['node']['page-title'] = array(
+      'name' => t('Page Title'),
+      'description' => t('The page title of the current node, as processed by the Page Title module.'),
+  );
 
   return $info;
 }
@@ -24,6 +28,7 @@ function page_title_tokens($type, $tokens, array $data = array(), array $options
   $replacements = array();
   $sanitize = !empty($options['sanitize']);
 
+  // Current page token.
   if ($type == 'current-page') {
     foreach ($tokens as $name => $original) {
       switch ($name) {
@@ -34,5 +39,18 @@ function page_title_tokens($type, $tokens, array $data = array(), array $options
     }
   }
 
+  // Node token.
+  if ($type == 'node' && !empty($data['node'])) {
+    $node = $data['node'];
+    foreach ($tokens as $name => $original) {
+      switch ($name) {
+        case 'page-title':
+          $title = !empty($node->page_title) ? $node->page_title : $node->title;
+          $replacements[$original] = $sanitize ? check_plain($title) : $title;
+          break;
+      }
+    }
+  }
+
   return $replacements;
 }
